### PR TITLE
Portfolio template

### PR DESCRIPTION
**Pull request by tahseenzahra**
## Portfolio Mobile Version template

This pull request contains 
-Interacctive Contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -201,15 +202,34 @@ <h3 class="CSlist-title">
...
+      <p class="poppins-font footer-p">
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+      </p>
+      <form action="https://formspree.io/f/xwkydobk" id="form" method="post">
+        <input class="form-field" type="text" id="username" name="username" maxlength="30"
+        placeholder="Full Name" required>
+        <input class="form-field" type="email" id="user-email" name="user-email" placeholder="Email Address" required>
+        <textarea class="form-field" name="message" maxlength="500" rows="7" cols="70" id="form-msg" placeholder="Enter text here." required></textarea>
+        <button type="button" class="poppins-font button" id="footerbtn">Get It Touch</button>
+      </form>
```
Comment (mwanawabangona) [inline index.html]: - [ ] `required`
- [ ] Amazing job you have implementing the form so far.
- [ ] please make sure all the validation are working
- [ ] also making sure the form is posting after pressing the submit button. please see screenshots below

`if your posting is working fine you will be able to get this message`
Screenshot from 2022-07-07 12-58-36

`amd if your validation is working you should be able to get such messages`
Screenshot from 2022-07-07 13-00-42

please see the [form spree](https://formspree.io/html/) for further configuration

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
## Status: Changes Required

 ## Hello there @tahseenzahra  👋🏽

###  You have done an amazing job so far! 👏🏽 👏🏽


There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand:


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Solid pull request scroll
* ✅  Good use of flexbox 🧐
* ✅  Good quality commits

## Required Changes
 _**Check the comments under the review.**_

- [ ] `required`
- [ ] please add a descriptive Pr Not only are PR descriptions helpful context for your team, they also train you to more concisely and effectively communicate, which is commonly an undervalued skill. Improving your communication skills will make you a better programmer and teammate.

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @mwanawabangona in your question so I can receive the notification.
>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
>
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (tahseenzahra) [reply index.html]: @mwanawabangona Thanks for reviewing my code. I have fixed the issue now

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:

Your project is complete. There is nothing else to say about your project. :+1:
Go ahead and merge it. :muscle:

## Highlights :1st_place_medal:
:heavy_check_mark: Following LInter
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following Requirements
:heavy_check_mark: Great Validations

### [Optional]
- [ ] Please use a descriptive title for your pull request. You are using **Portfolio template** so you can use **Portfolio: add a contact form and finish the mobile version.** Your pull request title tells what is happening in this pull request :thinking:
- [ ] Please use a descriptive summary for your pull request. Your pull request summary describes completely what is happening in this pull request :blush:
- [ ] Please use a descriptive branch name. You are using **portfolio-template** so you can use **Portfolio-contact-form** Your pull branch name tells what is happening in this branch :thinking:

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! clap clap clap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**